lib/package-1-12-themes.js

```diff
@@ -21,7 +21,20 @@ function Package( files, runtime ) {
 	Package1_12.apply( this, arguments );
 }
 
-extend( Package.prototype, Package1_12.prototype );
+// Copy some properties from the Package1_12's prototype
+[
+	"AUTHORS.txt",
+	"LICENSE.txt",
+	"images"
+].concat(
+	Object.keys(Package1_12.prototype)
+		.filter( function( property ) {
+			return /\.css$/.test( property );
+		} )
+).forEach( function( prop ) {
+	Package.prototype[ prop ] = Package1_12.prototype[ prop ];
+});
```

Comment:
It wouldn't be hard, but it would be more error prone. This says to copy all CSS files, regardless of which files may exist.